config/configtelemetry/configtelemetry.go

```diff
@@ -95,6 +116,10 @@ type TelemetrySetting struct {
 	//  - "normal" adds some other indicators on top of basic.
 	//  - "detailed" adds dimensions and views to the previous levels.
 	MetricsLevelStr string `mapstructure:"metrics_level"`
+	// MetricsAddr is the [address]:port that metrics exposition should be bound to.
+	MetricsAddr string `mapstructure:"metrics_addr"`
+	// MetricsPrefix is the prefix added to metrics generated by the collector.
+	MetricsPrefix string `mapstructure:"metrics_prefix"`
```

Comment:
Why do you need these values in this struct?

Reply:
Better yet, why do we need this struct?  It is unused.